README.rst

```diff
@@ -113,6 +115,9 @@ Options
    to ``secure``, preventing it from being sent over plain ``http``.
 -  ``session_cookie_http_only``, default ``True``, set the session
    cookie to ``httponly``, preventing it from being read by JavaScript.
+-  ``session_cookie_samesite``, default ``Lax``, set this to ``Strict`` to prevent the cookie from being sent by the browser to the target site in all cross-site browsing context, even when following a regular link.
```

Comment:
Can you add a link to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#SameSite ?

Reply:
Done!